content-repo/extra-docs/articles/Email-Communication-Readme.md

```diff
@@ -0,0 +1,174 @@
+
+### Automations
+There are 3 Automations in this pack.
+* [DisplayEmailHtml](https://xsoar.pan.dev/docs/reference/scripts/display-email-html): Displays the original email in HTML format in the incident layout.
+
+* [PreprocessEmail](https://xsoar.pan.dev/docs/reference/scripts/preprocess-email): Pre-process script for Email Communication layout. This script checks if the incoming email contains an Incident ID to link the mail to an existing incident, and tags the email as "email-thread". 
+
+* [SendEmailReply](https://xsoar.pan.dev/docs/reference/scripts/send-email-reply): Sends the email reply. 
+
+### Classifiers
+There are 4 Classifiers in this pack. When you configure an instance of the Gmail integration or EWS V2 integration, use the following classifiers.
+
+* **EWS - Classifier - Email Communication**:  Classifies EWS email messages.
+* **EWS - Incoming Mapper - Email Communication**:  Maps incoming EWS email message fields. 
+* **Gmail - Classifier - Email Communication**:   Classifies Gmail email messages. 
+* **Gmail - Incoming Mapper - Email Communication**: Maps incoming Gmail email message fields.
+
+### Incident Fields
+There is 1 incident field - **Add CC To Email**. 
+
+### Incident Types
+There is 1 incident type - **Email Communication**.
+
+### Layout
+There is 1 layout - **Email Communication** 
+
+There are 3 interactive sections in which you can specify 1 or more email addresses to add as a CC to the email, create the body of the email, and add attachments. The remaining 3 sections are for viewing the original email and all the email communications and attachments associated with the incident.
+
+
+
+![Layout](https://raw.githubusercontent.com/demisto/content/84e7bc89c8757544804540e6711d4b9aba210ec1/Packs/EmailCommunication/doc_files/Email_Communication_layout.png)
+
+You can use the layout as-is for email communication. It can also be used for new incident types by adding the **Email Communication** layout to an incident type. See [Add the Email Communication Layout to an Incident Type](#add-the-email-communication-layout-to-an-incident-type) for details. 
+ 
+>**Important:** In order to add CC recipients or an attachment to the email reply, you must select the *Show empty fields* checkbox. 
+ 
+| Layout sections | Description |
+|------------------ | ------------- |
+| Add CC to email | Add 1 or more CC recipients to the email as a comma-separated list of email addresses.  |
+| Message body | Write the body of the email reply. It is sent as an email and added to the War Room entry. |
+| Attachments | Add attachments to the email reply.  |
+| Email thread | Displays the entire email thread including the original email and all email replies. |  
+| Original Email HTML | Displays in HTML format the original email that opened the incident. |  
+| Mail Attachments | Displays the metadata of the email attachments. |  
+ ---
+
+> Warning: There is no notification for failed emails/addresses.
```

Comment:
what does this mean? @altmannyarden @ShirleyDenkberg

Reply:
@ShirleyDenkberg, please remove this line. In other mail clients, you don't get on error about the wrong email address (you will get another email from the mail server that the email address wasn't found)

Reply:
We will reword it a bit, but this should stay.
The whole point of this pack is to replace having to use a mail client as part of an incident. If we are not notifying users of an email that wasn't sent successfully, we're requiring them to rely on the mail client still.

These are the exact kinds of information that lead to CS tickets.

Reply:
I will reword this as follows:
Note: If an email cannot be sent to a specified address, no notification will appear in Cortex XSOAR.